README.md

```diff
@@ -160,6 +199,44 @@ FirebaseAnalytics.logEvent({
     items: [{ name: "Kittens" }],
   },
 });
+
+/**
+ * Platform: Web/Android/iOS
+ * Sets whether analytics collection is enabled for this app on this device.
+ * @param name - enabled - boolean true/false
+ * @returns void
+ */
+FirebaseAnalytics.setCollectionEnabled({
+  enabled: false,
+});
+
+/**
+ * Platform: Web/Android/iOS
+ * Deprecated - use setCollectionEnabled() instead
```

Comment:
Could probably use `@deprecated` for this: https://jsdoc.app/tags-deprecated.html